be/src/runtime/block_spill_manager.cpp

```diff
@@ -21,16 +21,28 @@
 #include <boost/uuid/uuid_io.hpp>
 #include <random>
 
+#include "util/file_utils.h"
+
 namespace doris {
+static const std::string BLOCK_SPILL_TMP_DIR = "spill";
 BlockSpillManager::BlockSpillManager(const std::vector<StorePath>& paths) : _store_paths(paths) {}
 
+Status BlockSpillManager::init() {
+    for (const auto& path : _store_paths) {
+        std::string dir = path.path + "/" + BLOCK_SPILL_TMP_DIR;
+        if (!FileUtils::check_exist(dir)) {
+            RETURN_IF_ERROR(FileUtils::create_dir(dir));
+        }
+    }
+    return Status::OK();
+}
 Status BlockSpillManager::get_writer(int32_t batch_size, vectorized::BlockSpillWriterUPtr& writer) {
     int64_t id;
     std::vector<int> indices(_store_paths.size());
     std::iota(indices.begin(), indices.end(), 0);
     std::shuffle(indices.begin(), indices.end(), std::mt19937 {std::random_device {}()});
 
-    std::string path = _store_paths[indices[0]].path;
+    std::string path = _store_paths[indices[0]].path + "/" + BLOCK_SPILL_TMP_DIR;
```

Comment:
There will be some garbage if be restart during spill to disk.
so that we could add a timestamp prefix here.
Then  add a gc logic to clear all folders before current timestamp.